### PR TITLE
Fixes #2519: removed telemetry values from available selection in Glo…

### LINKED ISSUE #2519: Companion 2.1.99 crashes when I open general settings
companion-crash
Companion 2.1.99 crashes when I open general settings.
It doesn't do this when it's empty...

It does so when I open General settings after opening an eepe or reading the settings from my radio.
I just tested it using remote desktop on my machine at work and it does the same.
It seems to be related to my radio's setup, so I will attach the eepe
20jul2015 eepe

### PR DESCRIPTION
**Pull request by projectkk2glider**
…bal Functions for "Play Value"

Closes #2519

### CODE REVIEW

File: companion/src/helpers.cpp [hunk 1]
```diff
@@ -681,7 +681,7 @@ void populateSourceCB(QComboBox *b, const RawSource & source, const GeneralSetti
     }
     else {
       for (int i=0; i<(flags & POPULATE_TELEMETRYEXT ? TELEMETRY_SOURCES_STATUS_COUNT : TELEMETRY_SOURCES_COUNT); i++) {
-        if (i==TELEMETRY_SOURCE_RSSI_TX && IS_TARANIS(board))
+        if (i==TELEMETRY_SOURCE_RSSI_TX && IS_TARANIS(board))  // TODO this condition is never true since else clause is never taken for ARM boards!!!
```
Comment (projectkk2glider) [inline companion/src/helpers.cpp]: This is just something I found while working nearby. I think the whole condition it is obsolete and could be removed.

**Comment by bsongis** [comment]
Thanks a lot! Would you port it to 'next' ? For users on 2.1.99 of course ;)

**Comment by projectkk2glider** [comment]
Way ahead of you :smile:  done.

**Comment by bsongis** [comment]
oh ok, sorry :)

**Comment by projectkk2glider** [comment]
No, no, don't be sorry. It is easy to forget something. The reminders are always welcome.

**Pull request merged**